docs/api-docs/webhooks/webhook-events.mdx

```diff
@@ -554,9 +554,156 @@ Payload objects with the following scopes take the form that follows:
+
+* `store/priceList/created`
+* `store/priceList/updated/` 
+* `store/priceList/activated/`
+* `store/priceList/deactivated/`
+* `store/priceList/deleted/`
+
+```json filename="Example price list created payload object" showLineNumbers copy
+{
+  "scope": "store/priceList/created",
+  "store_id": "1001197568",
+  "data": {
+    "price_list_id": 5
+  },
+  "hash": "e0c298b8097a6a2f39d17e593a9b360f5b2fef7d",
+  "created_at": 1709327895,
+  "producer": "stores/{store_hash}"
+}
+```
+The following price lists batch webhook events fire in response to actions that affect price lists:
+
+| Name / Scope | Description |
+|:-------------|:------------|
+| `store/priceLists/deleted/` | Fires when price list are deleted. | 
+
+Payload objects with the following scopes take the form that follows:
+* `store/priceLists/deleted/`
+
+```json filename="Example price lists deleted payload object" showLineNumbers copy
+{
+  "scope": "store/priceLists/deleted",
+  "store_id": "1001197568",
+  "data": {
+    "price_list_ids": [
+      5
+    ]
+  },
+  "hash": "e0c298b8097a6a2f39d17e593a9b360f5b2fef7d",
+  "created_at": 1709327895,
+  "producer": "stores/{store_hash}"
+}
+```
+
+
+## Price list records
+
+The following price lists records webhook events fire in response to actions that affect a specific price list record:
+
+| Name / Scope | Description |
+|:-------------|:------------|
+| `store/priceList/record/created/` | Fires when a price list record is created. | 
+| `store/priceList/record/updated/` | Fires when a price list record is updated. | 
+| `store/priceList/record/deleted/` | Fires when a price list record is deleted. | 
+
+
+Payload objects with the following scopes take the form that follows:
+
+* `store/priceList/record/created`
+* `store/priceList/record/updated/` 
+* `store/priceList/record/deleted/`
+
+```json filename="Example price list record created payload object" showLineNumbers copy
+{
+  "scope": "store/priceList/record/created",
+  "store_id": "1001197568",
+  "data": {
+    "price_list_id": 1,
+    "variant_id": 179,
+    "currency": "USD"
+      },
+  "hash": "0424a15f158bbfe7277c5f84f9c55a1d4a762e60",
+  "created_at": 1709327895,
+  "producer": "stores/{store_hash}"
+}
+```
+
+The following price list records batch webhook events fire in response to actions that affect price list records:
+
+| Name / Scope | Description |
+|:-------------|:------------|
+| `store/priceList/records/created/` | Fires when price list records are created. | 
+| `store/priceList/records/updated/` | Fires when price list records are updated. | 
+| `store/priceList/records/deleted/` | Fires when price list records are deleted. | 
+
+Payload objects with the following scopes take the form that follows:
+* `store/priceList/records/created`
+* `store/priceList/records/updated/`
+* `store/priceList/records/deleted/`
+
+```json filename="Example price list records created payload object" showLineNumbers copy
```

Comment:
```suggestion
```json filename="Example payload object for creating price list records" showLineNumbers copy
```